src/boot_arm.c

```diff
@@ -187,12 +196,12 @@ void isr_reset(void) {
         dst++;
     }
 
-#if !defined(PLATFORM_stm32l5) || (__ARM_FEATURE_CMSE == 0) || (__ARM_FEATURE_CMSE != 3U)  
```

Comment:
Removed this guard as I believe it is handled by `WOLFBOOT_NO_MPU` making `mpu_init` a nop.

Reply:
OK, this was here because our MPU code is not compatible with Cortex-M33 which has a different MPU controller that we don't support.